DZ2/src/datecalc.js

```diff
@@ -0,0 +1,43 @@
+import { DateTime } from "./luxon.js";
```

Comment:
тк у нас теперь есть npm то нужно ставить luxon через npm
и подключать его здесь уже немного иначе
```suggestion
import { DateTime } from "luxon";
```

Reply:
не уверен что сейчас у тебя вообще рабоатет luxon(